tests/Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM cartesi/sdk:0.6.0
+FROM cartesi/sdk:0.9.0
 
 RUN apt-get update
-RUN apt-get install -y build-essential luarocks lua5.4-dev git
+RUN apt-get install -y \
+  build-essential \
+  git \
+  lua5.4-dev \
+  luarocks
```

Comment:
Why not fix these versions as well?

Reply:
Will do.

Reply:
sent a fixup at [c41fcc6](https://github.com/crypto-bug-hunters/bug-buster/pull/141/commits/c41fcc69c66afd0395878adb686a26218198a945)